readme.md

```diff
@@ -23,11 +23,26 @@ During development, whippet can be used in conjunction with [wpc](https://github
 
 ### Install whippet
 
-To install Whippet, clone this directory and install its dependencies:
+To install Whippet, clone this directory:
 
 ```
 $ git clone --recurse-submodules https://github.com/dxw/whippet.git
+```
+
+Add it to your path:
+```
+$ cp whippet/bin/whippet /usr/local/bin/whippet
```

Comment:
This won't work. The composer deps are complied into `vendor.phar`, not `bin/whippet`.

Try this instead: `sudo ln -s `pwd`/whippet/bin/whippet /usr/local/bin/whippet`

Reply:
Ah, the way I moved the binary out might explain my original problem that got me thinking I needed composer locally to run whippet. 👍